library/core/src/intrinsics/fallback.rs

```diff
@@ -171,6 +171,9 @@ macro_rules! impl_funnel_shifts {
     ($($type:ident),*) => {$(
         #[rustc_const_unstable(feature = "core_intrinsics_fallbacks", issue = "none")]
         impl const FunnelShift for $type {
+            #[ferrocene::annotation(
+                "This function is indirectly but thoroughly tested through the `test_funnel_shift`, `test_funnel_shl_overflow` and `test_funnel_shifts_runtime` tests in `coretests::num`. The fact that is shown as uncovered is a known problem in our coverage tooling."
+            )]
```

Comment:
Maybe the reason this is not covered is because this is a fallback in case a specific target doesn't have the `FunnelShift` operation as an intrinsic.

That being said, I remember adding tests for other fallbacks as those are accessible once the right features are enabled in `coretests`.